packages/browser-destinations/src/destinations/adobe-target/triggerView/index.ts

```diff
@@ -6,7 +6,8 @@ import { setPageParams } from '../utils'
 
 const action: BrowserActionDefinition<Settings, Adobe, Payload> = {
   title: 'Trigger View',
-  description: 'Record a view',
+  defaultSubscription: 'type = "page"',
+  description: 'Send page-level data to Adobe Target.',
   platform: 'web',
   fields: {
     viewName: {
```

Comment:
The description here has a typo in the word `the`. Need to remove the "w" at the end :)

Reply:
Also we use snake case for the track event fields above but camel case for these fields, i.e. viewName. Should we standardize? Looks like other actions destinations use snake cake so maybe we should update all the fields in this action to also be snake cased?

```suggestion
    view_name: {
```

Reply:
Good catch! Looks like the convention is to have camel case. I'll switch the variables to that format instead.